AvatarChatbot/docker/gaudi/README.md

```diff
@@ -0,0 +1,226 @@
+# AvatarChatbot Application
+
+The AvatarChatbot service can be effortlessly deployed on either Intel Gaudi2 or Intel XEON Scalable Processors.
+
+## 🚀 Megaservice flow
+
+```mermaid
+flowchart LR
+    subgraph AvatarChatbot
+        direction LR
+        A[User] --> |Input query| B[AvatarChatbot Gateway]
+        B --> |Invoke| Megaservice
+        subgraph Megaservice["AvatarChatbot Megaservice"]
+            direction LR
+            subgraph AudioQnA["AudioQnA"]
+                direction TB
+                C((ASR<br>3001)) -. Post .-> D{{whisper-service<br>7066}}
```

Comment:
possible to mention the ASR implementation like ASR - whisper?

Reply:
Fixed in the updated mermaid flowchart